pages/product-buys-sale/mixins_product-buys-sale.pug

```diff
@@ -0,0 +1,29 @@
+mixin productItem(obj)
+    each n in obj
```

Comment:
đặt tên biến có nghĩa vào e, item in items chẳng hạn

Reply:
ok rồi a